publish-nuget/action.yml

```diff
@@ -86,9 +86,9 @@ runs:
     - name: Package
       shell: bash
       run: |
-        packables=$(grep -rl '<Import Project="\.\.\/Packable\.Projects\.props" \/>' ./src/**/*.csproj)
-        for project in $packables; do echo "packaging project $project" && \
-          dotnet pack $project --no-build --configuration ${{ steps.comp-mode.outputs.compilationMode }} \
+        for f in $(find . -name "*.sln"); do echo "packaging solution $f" && \
```

Comment:
I think it is OK, but what I liked about the other way of doing is that the user has to see that a `Packable.Projects.props` file is needed, and will likely find the good one, with icons, license, etc.
When simply looking for 1 property the package can come with a bit less care given to it... no big deal honestly

Reply:
Right. The thing is that it had the relative directory inside the search, and the directory level isn't the same across repositories (old folder structure X new folder structure). That's why I changed this. I'll check if we can check if logo and license were applied in the package, so we can avoid this to be published.

Reply:
Could we have something similar to `ReadmeDocumentationUpdater` where we have a test that checks we have good package info? Instead of doing it in the pipeline.

Reply:
I think so, but it would be only in the target project, and we should create something to validate that.
I changed this to get all csproj with that file mentioned in the XML. Please check it again :)

Reply:
> Could we have something similar to `ReadmeDocumentationUpdater` where we have a test that checks we have good package info? Instead of doing it in the pipeline.

Added a new epic in Jira for this: https://trakx.atlassian.net/browse/TECH-56